plugin/src/main/resources/io/jenkins/plugins/coverage/model/CoverageBuildAction/summary.jelly

```diff
@@ -4,38 +4,103 @@
+                          <j:set var="color" value="#9d1212"/>
+                        </j:when>
+                        <j:otherwise>
+                          <j:set var="color" value="#0a5f0e"/>
+                        </j:otherwise>
+                      </j:choose>
+                      <span style="color: ${color}">(${it.formatDelta(metric)})</span>
+                    </span>
+                  </j:if>
+                </li>
+              </span>
+            </j:forEach>
+          </ul>
+        </li>
+        <j:choose>
+          <j:when test="${it.hasChangeCoverage()}">
+            <li>
+              <b>
+                <a href="coverage/#changeCoverageTree">Change Coverage:</a>
+              </b>
+              <ul>
+                <j:forEach var="metric" items="${metrics}">
+                  <span id="change-coverage" title="The code coverage of only modified lines of code">
+                    <li>${it.formatChangeCoverage(metric)}
+                      <j:if test="${it.hasChangeCoverageDifference(metric)}">
+                        <span id="change-coverage-delta"
+                              title="The change code coverage delta against the reference build">
+                          <j:choose>
+                            <j:when test="${it.formatChangeCoverageDifference(metric).startsWith('-')}">
+                              <j:set var="color" value="#9d1212"/>
+                            </j:when>
+                            <j:otherwise>
+                              <j:set var="color" value="#0a5f0e"/>
+                            </j:otherwise>
+                          </j:choose>
+                          <span style="color: ${color}">(${it.formatChangeCoverageDifference(metric)})</span>
+                        </span>
+                      </j:if>
+                    </li>
+                  </span>
+                </j:forEach>
+                <span id="change-coverage-overview" title="The amount of modified lines which affect the change coverage">
+                  <li>${it.formatChangeCoverageOverview()}</li>
                 </span>
-              </j:if>
+              </ul>
             </li>
-          </span>
-        </j:forEach>
+          </j:when>
+          <j:otherwise>
+            <j:if test="${it.hasCodeChanges()}">
+              <span id="change-without-coverage">
+                <ul>
+                  <li><b>The code coverage has not been affected by any code changes.</b></li>
+                </ul>
+              </span>
+            </j:if>
+          </j:otherwise>
+        </j:choose>
+        <j:if test="${it.hasIndirectCoverageChanges()}">
+          <li>
+            <b>
+              <a href="coverage/#coverageChangesTree">Indirect Coverage Changes:</a>
+            </b>
+            <ul>
+              <j:forEach var="metric" items="${metrics}">
+                <span id="indirect-coverage-changes"
```

Comment:
Can you use the class `jenkins-tooltip` so that the tooltip uses the new tooltip styles? Best thing would be to install https://plugins.jenkins.io/design-library/ for reference.

Reply:
viewable online at https://weekly.ci.jenkins.io/design-library/Tooltips/